### ISSUE TITLE
the size of tenor a (9) must match the size of tenor b (10) at a non singleton dimension 3

### ISSUE THREAD

**Issue by shezhi** [lucidrains/bottleneck-transformer-pytorch#11]
Hello, I want to ask a question, the input feature map is 228 * 304, but here is an error, the size of tenor a (9) must match the size of tenor B (10) at a non singleton dimension 3.

**Comment by shezhi** [comment]
my code is this .
import torch
from torch import nn
from torchvision.models import resnet50
from bottleneck_transformer_pytorch import BottleStack
layer = BottleStack(
    dim = 1024,
    fmap_size = (15,19),        # set specifically for imagenet's 224 x 224
    dim_out = 2048,
    proj_factor = 4,
    downsample = True,
    heads = 4,
    dim_head = 128,
    rel_pos_emb = True,
    activation = nn.ReLU()
)
resnet = resnet50()
backbone = list(resnet.children())
model = nn.Sequential(
    *backbone[:7],
    layer
)

img = torch.randn(2, 3, 228, 304)
preds = model(img)
